force-app/main/default/lwc/note/note.js

```diff
@@ -1,147 +1,229 @@
-import { LightningElement, track, wire} from 'lwc';
+
+import { LightningElement, track } from 'lwc';
 import { ShowToastEvent} from 'lightning/platformShowToastEvent';
 import MyModal from 'c/myModal';
-import saveData from '@salesforce/apex/NoteController.saveData'
-import deleteNoteOnServer from '@salesforce/apex/NoteController.deleteNoteOnServer';
+import EditStiker from 'c/editStiker';
+import saveData from '@salesforce/apex/NoteController.saveData';
 import getSavedNotes from '@salesforce/apex/NoteController.getSavedNotes';
+import searchNotesByDate from '@salesforce/apex/NoteController.searchNotesByDate';
+import updateNote from '@salesforce/apex/NoteController.updateNote';
+import  deleteNoteOnServer from '@salesforce/apex/NoteController.deleteNoteOnServer';
 
 export default class Note extends LightningElement {
 
-    @track savedData= [];
+
+    @track saveNotes = [];
+    searchDate = '';
+    noteToUpdate;
+    searchNull;
+
+    
+    connectedCallback(){
+        this.loadSavedData();
+        console.log('Work');
+    }   
+
+    handleDateChange(event) {
+        this.searchDate = event.target.value;
+    }
+    
 
 
-    //1
-    @wire(getSavedNotes)
-    wiredSavedNotes({ error, data}){
-        console.log('Wire is called');
-        if (data) {
-            this.savedData = data;
-            console.log("Result: " + data);
-        } else if (error) {
+    async loadSavedData(){
+        console.log('Loading data from server...');
+        try{
+            const result = await getSavedNotes();
+            this.saveNotes = result;
```

Comment:
saveNotes sounds like verb. Please call variables like noun. Function/method names should be verbs